omod-common/src/test/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImplTest.java

```diff
@@ -637,12 +638,13 @@ public void getSearchHandler_shouldReturnSearchHandlerProvidingAllRequestParamet
 		
 		SearchHandler searchHandler1 = mock(SearchHandler.class);
 		SearchConfig searchConfig1 = new SearchConfig("default", "v1/concept", "1.8.*", new SearchQuery.Builder(
-		        "description").withRequiredParameters("source").withOptionalParameters("code").build());
+		        "description").withRequiredParameters(new SearchParameter("source", "some source"))
+		        .withOptionalParameters(new SearchParameter("code", "some code")).build());
 		when(searchHandler1.getSearchConfig()).thenReturn(searchConfig1);
 		
 		SearchHandler searchHandler2 = mock(SearchHandler.class);
 		SearchConfig searchConfig2 = new SearchConfig("conceptByMapping", "v1/concept", "1.8.*", new SearchQuery.Builder(
-		        "description").withRequiredParameters("source").build());
+		        "description").withRequiredParameters(new SearchParameter("source", "some source")).build());
```

Comment:
Yes. This test was failing, but it depends on changes made in SearchQuery, exactly on type of required and optionalParameters, so there I needed to change test.